quirks/password-rules.json

```diff
@@ -222,6 +222,9 @@
     "metlife.com": {
         "password-rules": "minlength: 6; maxlength: 20;"
     },
+    "microsoft.com": {
```

Comment:
Is there a page on microsoft.com where you can set a password? Or is it only live.com?

Reply:
Your rules look good, but in my experience, the domain I change passwords on is live.com. Were you able to change or initially set a password on microsoft.com?

Reply:
@3NCRY9T3R Can you answer my question about live.com?

Reply:
I'm not sure whether this is also true for for business accounts, but based on my experience with personal Microsoft accounts I'd agree with @rmondello that creating the account, changing the password and even logging-in happens on live.com.

Reply:
I can confirm that choosing to signin at https://www.microsoft.com/ redirects you to https://login.live.com/. This is also where the option to create an account seems to exist.